docs/migration.md

```diff
@@ -0,0 +1,26 @@
+# Migration guides
+
+## Generic migration
+Migration for the typical mod which doesn't usy anything special but minecraft forge and some library dependencies. 
+For core plugin, mixins, shadowing, access transformes, asm or etc. you'll need to do some extra steps. 
+If they are missing in this document - we will gladly receive your suggestions/contribution. 
+
+1. Copy and replace all files from template to your repository, but: // TODO: (maybe we should have a task in CI which attaches zip with all files but these, called "migration-bundle.zip" or smth)
```

Comment:
No idea how to do that, but it might be a nice idea

Reply:
I think you can do this with actions we already have, just add a task which will zip all the things we want to and add it to the release, smth like:

```
    - name: Gather migration template files
      run: zip build/migrations.zip * --exclude build.gradle,src,...
        
    - uses: "marvinpinto/action-automatic-releases@latest"
      with:
        repo_token: "${{ secrets.GITHUB_TOKEN }}"
        automatic_release_tag: "latest"
        prerelease: true
        title: "Latest Development Build"
        files: [build/libs/*.jar, build/migrations.zip]
```
This is a pseudo code, I don't remember how to exclude things in zip command and am not exactly sure how to specify files in the actions.

Reply:
We need to get this task in a separate file thought. Since the current file is supposed to be part of the pack.